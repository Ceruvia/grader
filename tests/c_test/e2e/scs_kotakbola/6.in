193 739 887
M 6
H 47
B 86
B 64
H 53
B 54
B 66
M 81
M 29
H 85
E
