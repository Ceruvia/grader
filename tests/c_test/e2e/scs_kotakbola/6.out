185 116 270
Ya
